src/parser/mod.rs

```diff
@@ -6043,6 +6043,20 @@ impl<'a> Parser<'a> {
             && dialect_of!(self is MySqlDialect | SQLiteDialect | DuckDbDialect | GenericDialect)
         {
             self.parse_optional_column_option_as()
+        } else if self.parse_keyword(Keyword::IDENTITY)
+            && dialect_of!(self is MsSqlDialect | GenericDialect)
+        {
+            let parameters = if self.expect_token(&Token::LParen).is_ok() {
```

Comment:
We can replace the `expect_token().is_ok()` with `consume_token()`?

Reply:
Updated.